pandas/tests/sparse/test_libsparse.py

```diff
@@ -596,22 +596,10 @@ def _check_case(xloc, xlen, yloc, ylen, eloc, elen):
 
         check_cases(_check_case)
 
-
-# too cute? oh but how I abhor code duplication
-check_ops = ['add', 'sub', 'mul', 'truediv', 'floordiv']
-
-
-def make_optestf(op):
-    def f(self):
-        sparse_op = getattr(splib, 'sparse_%s_float64' % op)
-        python_op = getattr(operator, op)
+    @pytest.mark.parametrize('opname',
```

Comment:
worth making a fixture in conftest?

Reply:
I think this is a pretty specific set of strings that wouldn't get used often.  But I didn't look at this closely, just saw that the existing formatting was crying out to be updated.

Reply:
ok, that's fine